File: project3/Team2/backend/src/main/java/bitc/full502/backend/dto/OrderDTO.java
package bitc.full502.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OrderDTO {
    private int orKey;         // 주문 번호
    private int agKey;         // 대리점 키
    private String orDate;     // 주문일
    private String orStatus;   // 처리 상태
    private String orReserve;  // 도착 예정일
    private String dvName;     // 배송기사 이름
    private String dvPhone;    // 배송기사 전화번호
    private int orTotal;       // 총액
}

File: project3/Team2/android/app/src/main/java/bitc/fullstack502/finalproject/OrderDTO.kt
package bitc.fullstack502.finalproject

data class OrderDTO(
    val orKey: Int,          // 주문번호
    val agKey: Int,          // 대리점 키 (로그인한 사용자 필터용)
    val orDate: String?,     // 주문일
    val orStatus: String?,   // 처리 상태
    val orReserve: String?,  // 도착 예정일
    val dvName: String?,     // 배송기사 이름
    val dvPhone: String?,    // 배송기사 전화번호
    val orTotal: Int         // 총액
)